src/main/kotlin/no/nav/syfo/client/pdl/PdlPersonResponse.kt

```diff
@@ -41,6 +51,7 @@ data class PdlPerson(
     val kontaktadresse: List<Kontaktadresse>?,
     val oppholdsadresse: List<Oppholdsadresse>?,
     val doedsfall: List<PdlDoedsfall>?,
+    val tilrettelagtKommunikasjon: List<PdlTilrettelagtKommunikasjon>?,
```

Comment:
Vi ville egentlig at denne ikke skulle være nullable, men tom liste hvis det ikke var noe der. Da fikk vi serialiseringsfeil. Det er jo slikt med `navn` på linje 48, så vet ikke helt hvorfor 🤷🏼‍♂️ 

Vi er ikke så godt kjent med GraphQL, men vil alle verdier som ikke gir "svar" bare være `null`? Og hvorfor gir ikke `navn` den samme feilen (kanskje fordi `navn` alltid er der i alle testene våre 🤔).

Reply:
Er det en greie med GraphQL at alle properties er av typen `List` ? 😊

Reply:
Ikke så kjent med GraphQL selv, men står litt her: https://pdl-docs.intern.nav.no/ekstern/index.html#_skjemadefinisjon. Sånn det ser ut [her](https://github.com/navikt/pdl/blob/master/apps/api/src/main/resources/schemas/pdl.graphqls#L158) så vil  ikke denne listen være `null` i responsen fra APIet 🤔

Reply:
@eirikdahlen Kan det være at den feilen dere fikk var pga first() i stedet for firstOrNull() ovenfor?

Reply:
Det burde funke med ikke nullable på denne hvis man tilpasser `PdlMock` til å håndtere det. Eller fikk dere feil ved testing mot det faktiske APIet?

Reply:
Ja var mot APIet, vi hadde ikke lest specen godt nok 👍🏼 Siste versjon funker i dev